### PR TITLE
Improved French version and create English version

### PR DESCRIPTION
**Pull request by octaveauger**
This does 2 out of the 3 things planned for the Pro Dashboard tutorial:
- Improve the French version (mostly by linking to the production account instead of sandbox)
- Add an English version, which should not appear on the UK site but on the EU and Irish sites. This will allow us to translate it to German.
- Next step: add a more detailed section on representing failed payments

@Ken-Lo can you check the English version? Also can you change it so the tutorial does not appear on the UK site? I'm not sure how I can remove it (I guess not have 'en' in the routes?)
@adeclosset can you check the French version?

### CODE REVIEW

**Comment by adeclosset** [comment]
Yes, all good. Should make sure the /be-fr version is up-to-date too.

**Comment by Ken-Lo** [comment]
@octaveauger Thanks for this! Will take a look at the English version when I'm in tomorrow morning and will get this merged :)

**Comment by octaveauger** [comment]
@adeclosset the Belgian version is part of the same content (you can see some parts where there's ifLocalSignup=false)
@Ken-Lo don't merge it yet, I am adding a new section as part of this branch

**Comment by Ken-Lo** [comment]
@octaveauger Awesome work :) I have proof-read/updated/clarified the English version... please let me know if you have any comments. There are some clarifications/rewordings which you might also want to translate back into French. Otherwise, more than happy to merge this into dev!

**Comment by octaveauger** [comment]
@Ken-Lo I've harmonised the French content and added a new section around payment failures and retries.
@jamesshedden I couldn't find a table style that would fit. Can you help us fix the vision of horror below?
@adeclosset fyi - feel free to take a look at the French retry page (no need to check the rest)

image

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        <ul className='list'>
+          <li>
+            <strong>The payment fails and can be retried</strong>: this is the case mainly for the following causes: insufficient funds
+            and refer to payer
+          </li>
+          <li>
+            <strong>The payment fails as well as the mandate</strong>: you should contact your customer to setup a new mandate
+          </li>
+          <li>
+            <strong>The payment is disputed</strong>: you should contact the customer to understand why they contested and only create
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: **The payment has been disputed by the customer.** You should contact the customer to understand why the payment has been disputed. You should only create a new payment once the dispute is resolved.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        <p className='para'>
+          When a payment fails, we will indicate the reason as well as a short description of its cause. There are 3 main types of failures:
+        </p>
+        <ul className='list'>
+          <li>
+            <strong>The payment fails and can be retried</strong>: this is the case mainly for the following causes: insufficient funds
+            and refer to payer
+          </li>
+          <li>
+            <strong>The payment fails as well as the mandate</strong>: you should contact your customer to setup a new mandate
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: **The payment has failed along with the mandate.** You should contact the customer to set up a new mandate.

Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: (if we're using the phrase "The payment fails as well as the mandate" in the dashboard, then we should keep your original version for consistency with that)

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  render() {
+    return (
+      <Translation locales={['en']} exclude={['en-GB']}>
+        <h2 className='u-text-heading-light u-text-m u-color-heading'>Payment retry</h2>
+        <p className='para'>
+          When a payment fails, we will indicate the reason as well as a short description of its cause. There are 3 main types of failures:
+        </p>
+        <ul className='list'>
+          <li>
+            <strong>The payment fails and can be retried</strong>: this is the case mainly for the following causes: insufficient funds
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: **The payment has failed and can be retried.** This normally happens when there is insufficient funds in your customer's bank account or when a "refer to payer" error was received.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        <p className='para'>
+          When possible, you can <strong>retry the payment</strong> by clicking "Retry". We will resubmit your payment to the banks
+          straight away. You can see the new charge date as well as follow the status of the payment in the same history as your initial
+          payment.
+        </p>
+
+        <img src='/images/faq/en/payment-retry.png' />
+
+        <p className='para'>
+          Please find below a full list of the payment failure causes and what they mean.
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: The following table shows a list of possible payment failure errors and an explanation for what the error message means:

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+            <strong>The payment fails as well as the mandate</strong>: you should contact your customer to setup a new mandate
+          </li>
+          <li>
+            <strong>The payment is disputed</strong>: you should contact the customer to understand why they contested and only create
+            a new payment once this is clarified
+          </li>
+        </ul>
+
+        <p className='para'>
+          When possible, you can <strong>retry the payment</strong> by clicking "Retry". We will resubmit your payment to the banks
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: Where it's possible to retry a payment, you'll see a "Retry" button in the top-right corner of the payment status page. If you click this button, we'll re-submit your payment immediately. You'll see a new charge date and you can follow the status of the retried payment in the payment timeline.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+              </th>
+              <th className='u-padding-Am'>
+                Explanation
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Insufficient funds</td>
+              <td className='comparison-table__cell'>The customer did not have funds available to make the payment.</td>
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: Your customer's bank account had insufficient funds available on the payment date.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Refer to payer</td>
+              <td className='comparison-table__cell'>This is often used by banks to obfuscate the failure cause: quite commonly
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: The 'refer to payer' error is often used by the banks when no clear reason is given for the failure. Quite commonly, it means insufficient funds. (how commonly?)

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+              indicates insufficient funds.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Bank account closed</td>
+              <td className='comparison-table__cell'>The customer's bank account was closed before the payment could be taken.
+              The mandate will be cancelled.</td>
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: Something like "The mandate and all future payments will be cancelled" (to make it totally clear what the impact of this is).

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Direct Debit not enabled</td>
+              <td className='comparison-table__cell'>The customer's bank account does not allows direct debit payments.
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: The customer's bank account does not allow Direct Debit payments to be taken. The mandate and all future payments will be cancelled.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+              The mandate will be cancelled.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Invalid bank details</td>
+              <td className='comparison-table__cell'>The bank account details used to set up the mandate for this payment were incorrect.
+              The mandate will be cancelled.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Mandate cancelled</td>
+              <td className='comparison-table__cell'>Either you cancelled the mandate or the client cancelled it at their bank.</td>
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: Either you or the customer has cancelled the mandate. The customer may have cancelled the mandate at their bank.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Authorisation disputed</td>
+              <td className='comparison-table__cell'>In SEPA, customers are entitled to a refund of payments they did not authorise
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: In the SEPA Direct Debit scheme, customers are entitled to a refund for any payments they did not authorise. They have 13 months from the date of the payment to claim a refund.

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+              within 13 months.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Refund requested</td>
+              <td className='comparison-table__cell'>In SEPA, customers are entitled to a refund for authorised transactions
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.en.js]: In the SEPA Direct Debit scheme, customers have an 8-week window where they're able to get a refund for authorised transactions.

**Comment by Ken-Lo** [comment]
@octaveauger Awesome work! A couple of small suggestions above on making the English version a little more clear :) :+1: from me otherwise!

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+export default class FaqProDashboardPaymentRetryFr extends React.Component {
+  displayName = 'FaqProDashboardPaymentRetryFr'
+
+  render() {
+    return (
+      <Translation locales='fr'>
+        <h2 className='u-text-heading-light u-text-m u-color-heading'>Représentation de paiement</h2>
+        <p className='para'>
+          Quand un paiement échoue, nous vous indiquerons la cause ainsi qu'une brève description. Il y a 3 types principaux d'échec :
```
Comment (adeclosset) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js]: nous vous indiquerons **le motif**

Comment (adeclosset) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js]: Il y a 3 types principaux de **rejets** :

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+        <p className='para'>
+          Quand cela est possible, vous pouvez <strong>représenter le paiement</strong> en cliquant "Retry". Nous resoumettrons votre
+          paiement aux banques tout de suite. Vous pouvez voir la nouvelle date de prélèvement ainsi que suivre le statut du paiement
+          dans le même historique que le paiement initial.
+        </p>
+
+        <img src='/images/faq/fr/payment-retry.png' />
+
+        <p className='para'>
+          Nous avons listé ci-dessous la liste complète des échecs de paiement et ce qu'ils signifient.
```
Comment (adeclosset) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js]: Nous avons **énuméré** ci-dessous

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+        </p>
+
+        <table className='u-size-full'>
+          <thead>
+            <tr>
+              <th className='u-padding-Am'>
+                Echec de paiement
```
Comment (adeclosset) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js]: Je préfère "rejet" - cette terminologie est plus souvent utilisée j'ai l'impression

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+            </tr>
+          </thead>
+          <tbody>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Insufficient funds (fonds insuffisants)</td>
+              <td className='comparison-table__cell'>Le client n'avait pas assez de fonds pour faire le paiement.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Refer to payer (contactez le client)</td>
+              <td className='comparison-table__cell'>C'est souvent utilisé par les banques pour masquer la cause de l'échec et
```
Comment (adeclosset) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js]: **Ce motif** est souvent...

File: app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+              indique fréquemment des fonds insuffisants.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Bank account closed (compte en banque clôturé)</td>
+              <td className='comparison-table__cell'>Le compte en banque du client a été clôturé avant que le prélèvement n'ait
+              lieu. Le mandat sera annulé.</td>
+            </tr>
+            <tr className='comparison-table__row'>
+              <td className='comparison-table__cell'>Direct Debit not enabled (prélèvement impossible)</td>
+              <td className='comparison-table__cell'>Le compte en banque du client ne permet pas les prélèvements. Le mandat
```
Comment (adeclosset) [inline app/pages/faq/pro-dashboard/payment-retry/faq-pro-dashboard-payment-retry.fr.js]: **,par exemple il est sur liste noire ou un compte d'épargne.**

**Comment by octaveauger** [comment]
@adeclosset @Ken-Lo all done and designed thx to @jamesshedden, merging now!

**Pull request merged**